### PR TITLE
Opta Local Changes

### PR DESCRIPTION
**Pull request by bigbitbus**
# Description

Changes to integrate Opta local

# Safety checklist
* [X ] This change is backwards compatible and safe to apply by existing users
* [ X] This change will NOT lead to data loss
* [ X] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
manual testing, tests tbd.

### CODE REVIEW

File: config/tf_modules/local-base/linkerd.tf [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  allowed_uses = [
+    "crl_signing",
+    "cert_signing",
+    "server_auth",
+    "client_auth"
+  ]
+}
+
+
+resource "helm_release" "linkerd" {
```
Comment (juandiegopalomino) [inline config/tf_modules/local-base/linkerd.tf:57]: so we're installing linkerd for them? I thought we decided it was too heavy and not useful for local workloads lol

**Review by juandiegopalomino — APPROVED** [review]
one tiny question, otherwise lgtm

File: config/tf_modules/local-base/linkerd.tf (hunk 1 above)
Comment (bigbitbus) [reply config/tf_modules/local-base/linkerd.tf:57]: Its disabled by default (does not install because of the flag)

Comment (juandiegopalomino) [reply config/tf_modules/local-base/linkerd.tf:57]: ok cool

**Pull request merged**